samples/java-customer-registry-quickstart/pom.xml

```diff
@@ -70,7 +70,7 @@
               <includeSourceInfoInDescriptorSet>true</includeSourceInfoInDescriptorSet>
               <descriptorSetFileName>user-function.desc</descriptorSetFileName>
               <checkStaleness>true</checkStaleness>
-              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
+              <protocArtifact>com.google.protobuf:protoc:3.15.0:exe:${os.detected.classifier}</protocArtifact>
```

Comment:
Better align it with Akka gRPC and use 3.19.2?

Reply:
Sounds good!